README.md

```diff
@@ -159,6 +159,13 @@ Platforms without FIPS compliance are also included in this jar, for compatibili
 > [!WARNING]
 > The classifier, and platforms with FIPS compliance are subject to change in the future.
 
+## GraalVM support
+
+Since version v0.29.20, GraalVM native image was supported. You can compile your application with AWS CRT in a GraalVM native image project without any additional configuration.
+
+Since version v0.30.12, GraalVM support was updated. Instead of packaging the JNI shared lib with native image as resource, the corresponding shared lib will be written to the same directory as the native image.
+In this way, it reduces the native image size around 30%, and avoids the extra loading time needed for extracting the JNI lib to the temporary path for load. No additional configuration needed.
+**Note**: the extracted JNI lib, `libaws-crt-jni.*` , will need to be exist in the directory containing the native image, if you move the native image to different directory.
```

Comment:
Is it possible to keep the shared-lib in resources, but ONLY package the 1 shared-lib that users need?

Reply:
https://www.graalvm.org/sdk/javadoc/org/graalvm/nativeimage/hosted/RuntimeResourceAccess.html#addResource(java.lang.Module,java.lang.String)

This seems to be the only way available to make the resource from code. And it's introduced since 22.3.
And we are stuck at 21.3.11 for jdk8.

And I think it's fine for user to be aware of the separate shared lib to be part of the resource they need to include for graalvm.

1. GraalVM is already an advanced use case, people need to know their stuff to use GraalVM.
2. For Lambda use case, you need extra resources (boostrap script) for GraalVM to work, the shared lib is just one extra resource for user.